packages/commonwealth/server/farcaster/frames/contest/contestCard.tsx

```diff
@@ -101,6 +101,9 @@ export const contestCard = frames(async (ctx) => {
       >
         Check Eligibility
       </Button>,
+      <Button key="install" action="link" target={getActionInstallUrl()}>
```

Comment:
what happens if the action is already installed? maybe we should hide the button or make it disabled?